fire/core.py

```diff
@@ -166,13 +172,16 @@ class FireExit(SystemExit):
 
   Contains the trace of the Fire program.
 
-  If not caught, then a FireExit will cause the program to exit with a non-zero
-  status code.
+  Attributes:
```

Comment:
sure - should I add class level variable like `trace = None` just to clarify to users that they can pull out the `trace` property to programmatically inspect on error?